addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10313,7 +10313,11 @@ msgctxt "#20244"
 msgid "Android Apps"
 msgstr ""
 
-#empty strings from id 20245 to 20249
+msgctxt "#20245"
```

Comment:
@topfs2 Yeah, I might have made that clearer ;)

@da-anda That's just what it is, shortcuts. I don't want to start hacking our settings to add shortcuts in a cross-platform way.

It's just a nicety w/o much added value, really. I implemented it only because a box I use hided some settings ;)
I have no problem to take it out if it proves controversial in any way.